backend/tests/Designer.Tests/_TestData/Authorization/Policies/Xacml/ssb_ra1000-01.json

```diff
@@ -0,0 +1,131 @@
+{
+  "Rules": [
+    {
+      "RuleId": "urn:altinn:example:ruleid:1",
+      "Description": "A rule giving user with role REGNA or DAGL and the app owner ssb the right to instantiate a instance of a given app of ssb/ra1000-01",
+      "Subject": [
```

Comment:
Shouldn't Subject also use an array of arrays like resource? Now it will not support AND of subjects.

Reply:
There was a discussion on this when talking about format. So it was voted down because of simplicity